### ISSUE TITLE
ShareProgress

### ISSUE THREAD

**Issue by hannahkane** [MozillaFoundation/Advocacy#916]
I believe the plan is to have the Share feature share the homepage, rather than the particular video, at least to start (will file a separate ticket to potentially iterate on that).

So the copy for Sharing (email, FB, Twitter) should reference the homepage for now.

**Comment by HPaulJohnson** [comment]
Share copy here: 

https://docs.google.com/a/mozilla.com/document/d/14VQ2vvdxgnX0RvTIIiKoUMyB-sucuvBYnqdk1cpts7E/edit?usp=sharing

I kept it generic for now but could easily be video specific by dropping in a unique URL.

**Comment by HPaulJohnson** [comment]
we should create an image to accompany the Facebook share. @lovegushwa maybe something you can tackle when you get back?

**Comment by hannahkane** [comment]
assigning @ScottDowne to implement Share and @lovegushwa to create the image

**Comment by ScottDowne** [comment]
Just waiting on the images, and Paul is going to go in later and add test variants. Right now it's just 1 for each.

It does share the home page.

**Comment by HPaulJohnson** [comment]
@ScottDowne sounds like Jordan sent you the image for this. I just need a URL where it's hosted to update the SP page and buttons.
